build.sbt

```diff
@@ -916,9 +932,35 @@ lazy val liteEmbeddedDeploymentManager = (project in lite("embeddedDeploymentMan
   settings(assemblyNoScala("lite-embedded-manager.jar"): _*).
 
   settings(
-    name := "lite-embedded-deploymentManager",
+    name := "nussknacker-lite-embedded-deploymentManager",
   ).dependsOn(liteEngineKafkaRuntime, deploymentManagerApi % "provided", liteKafkaComponents % "test", testUtil % "test", kafkaTestUtil % "test")
 
+lazy val buildAndImportRuntimeImageToK3d = taskKey[Unit]("Import runtime image into k3d cluster")
+
+lazy val liteK8sDeploymentManager = (project in lite("k8sDeploymentManager")).
```

Comment:
let's add it to nussknacker-dist and ui/provided? Or TODO to do so, if you think it's not ready yet ;)